vendor/github.com/portworx/sched-ops/k8s/k8s.go

```diff
@@ -162,6 +162,10 @@ type DeploymentOps interface {
 	DescribeDeployment(name, namespace string) (*apps_api.DeploymentStatus, error)
 	// GetDeploymentsUsingStorageClass returns all deployments using the given storage class
 	GetDeploymentsUsingStorageClass(scName string) ([]apps_api.Deployment, error)
+	// UpdateDeployment updates the deployment (scale for now, can be extended to other things)
```

Comment:
do a pull in sched ops to get the GetDeployment method that was recently added

Reply:
Done.